### PR TITLE
items: complete the data model

### PR DESCRIPTION
**Pull request by BadrAly**
In order to complete the item data model, the following fields are
added to the JSON schema:

* url
* pac_code
* price
* _masked
* legacy_checkout_count
* legacy_circulation_rules

Co-Authored-by: Aly Badr <aly.badr@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by iGormilhit — APPROVED** [review]

**Review by benerken — APPROVED** [review]

File: rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json [hunk 1]
```diff
@@ -552,6 +556,133 @@
...
+        },
+        "floats": {
+          "title": "Item floats",
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "_masked": {
+      "title": "Masked",
```
Comment (pronguen) [inline rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json:674]: Add description: "A masked resource is visible in the professional interface, but not in the public interface."

File: rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json [hunk 2]
```diff
@@ -552,6 +556,133 @@
...
+          "cssClass": "editor-title"
+        },
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "pac_code": {
+      "title": "Conservation and preservation policy",
```
Comment (pronguen) [inline rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json:583]:
```suggestion
      "title": "PAC code",
```

And add a description: "Preservation and conservation policy"

File: rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json [hunk 3]
```diff
@@ -552,6 +556,133 @@
           }
         }
       }
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "placeholder": "Example: http://server.mediatheque.ch/CH-000019-X:223094.file",
```
Comment (pronguen) [inline rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json:563]: This is not useful. No placeholder was added for field `electronicLocator` in the document ([see here](https://github.com/rero/rero-ils/blob/dev/rero_ils/modules/documents/jsonschemas/documents/document_electronic_locator-v0.0.1.json))

```suggestion
```

File: rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json [hunk 4]
```diff
@@ -552,6 +556,133 @@
...
+          "minimum": 0,
+          "default": 0
+        },
+        "allow_request": {
+          "title": "Allow requests",
+          "type": "boolean",
+          "default": true
+        },
+        "floats": {
+          "title": "Item floats",
```
[the remark this answers is not in the document]
Comment (BadrAly) [reply rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json:667]: It is very specific to Virtua: check the `Min Floating Item Count Threshold` and `Max Floating Item Count Threshold` in the Virtua profiler: Circulation user guide.
If I recall correctly, `RBNJ` have asked to migrate this field ?

[the remark this answers is not in the document]
Comment (pronguen) [reply rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json:666]: Yes, Neuchâtel asked for it. I just wonder how we will need to translate this 😅 So if you have an idea of a better word, don't hesitate.

File: rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json (hunk 2 above)
Comment (pronguen) [reply rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json]: @BadrAly Here we should add the description "Preservation and conservation policy"

**Review by pronguen — APPROVED** [review]

**Review by rerowep — APPROVED** [review]

**Pull request merged**